src/CP/variables/IntVarView.jl

```diff
@@ -84,6 +84,7 @@ Return the "true" variable behind `x`.
 """
 rootVariable(x::IntVarView) = rootVariable(x.x)
 
+id(x::IntVarView) = id(x.x)

```

Comment:
This isn't coherent, as x already has a field `id`
To get the parent id use `rootVariable(x).id`